mangaki/mangaki/management/commands/morphing.py

```diff
@@ -0,0 +1,68 @@
+from django.core.management.base import BaseCommand, CommandError
+
+import json
+from mangaki.utils.values import tags_definition
+from math import floor
+import numpy as np
+
+posters = json.load(open('../fixtures/mangaki_i2v.json'))
+nb_tags = len(tags_definition)
+index_posters = [.0] * len(posters)
+i = 0
+for poster in posters:
+	index = int(poster[:len(poster)-4])
+	index_posters[i] = index
+	i += 1
+poster_tags = np.zeros((max(index_posters)+1, nb_tags))
+for poster in posters:
+	index = int(poster[:len(poster)-4])
+	for tag, value in posters[poster]['general']:
+		poster_tags[index][tags_definition[tag]] = value
+
+def distance(a, b):
+	dist = (b-a)**2
+	return np.sqrt(np.sum(dist))
+
+def dist2seg(point, a, segment):
+	seg_norm = np.sqrt(np.dot(segment, segment))
+	current = point - a
+	projection = np.dot(current, segment)/seg_norm
+	dist2seg = distance(a, point) - projection**2
```

Comment:
It may explain:

    RuntimeWarning: invalid value encountered in sqrt
      return np.sqrt(dist2seg)

😛